Client/src/Pages/Settings/index.jsx

```diff
@@ -1,9 +1,978 @@
+                  Delete account
+                </Typography>
+                <Typography variant="h5" component="p">
+                  Note that deleting your account will remove all data from our
+                  system. This is permanent and non-recoverable.
+                </Typography>
+                <Box sx={{ mt: theme.spacing(1) }}>
+                  <Button
+                    level="error"
+                    label="Delete account"
+                    onClick={() => setIsOpen(true)}
+                    sx={{
+                      fontSize: "13px",
+                      "&:focus": {
+                        outline: "none",
+                      },
+                    }}
+                  />
+                </Box>
+              </Stack>
+            </div>
+          </form>
+          <Divider
+            aria-hidden="true"
+            width="0"
+            sx={{ marginY: theme.spacing(6.25) }}
+          />
+          {/* !!! - All save buttons are tied to the same state */}
+          {/* TODO - Implement Save Profile function */}
+          <Stack direction="row" justifyContent="flex-end">
+            <Box width="fit-content">
+              <ButtonSpinner
+                level="primary"
+                label="Save"
+                onClick={handleSaveProfile}
+                isLoading={isLoading}
+                loadingText="Saving..."
+                sx={{
+                  paddingX: "40px",
+                  height: "fit-content",
+                  fontSize: "13px",
+                  "&:focus": {
+                    outline: "none",
+                  },
+                }}
+              />
+            </Box>
+          </Stack>
+        </TabPanel>
+        <TabPanel
+          value="1"
+          sx={{ padding: "0", marginTop: theme.spacing(6.25), width: "100%" }}
+        >
+          <form className="edit-password-form" noValidate spellCheck="false">
+            <div className="edit-password-form__wrapper">
+              <AnnouncementsDualButtonWithIcon
+                icon={<InfoOutlinedIcon style={{ fill: "#344054" }} />}
+                subject="SSO login"
+                body="Since you logged in via SSO, you cannot reset or modify your password."
+              />
+            </div>
+            {Object.entries(passwordConfig).map(([key, field]) => (
+              <div className="edit-password-form__wrapper" key={key}>
+                <Stack
+                  direction="column"
+                  gap="8px"
+                  sx={{ flex: 1, marginRight: "10px" }}
+                >
+                  <Typography variant="h4" component="h1">
+                    {field.label}
+                  </Typography>
+                </Stack>
+                {/* TODO - Refactor Password Input Component */}
+                {field.type === "input" ? (
+                  <FormControl sx={{ flex: 1, minWidth: theme.spacing(30) }}>
+                    <OutlinedInput
```

Comment:
We have components for dealing with Text and Password inputs implemented already, for consistency's sake we should use those throughout the app.

Please see the register and login components to see examples of their usage.

Reply:
Yeah I somehow missed those, will look over it on my next pr.